Week3/README.md

```diff
@@ -10,26 +10,32 @@ In week three we will discuss the following topics:
 • Functions
 ```
 
+
+> :boom: IMPORTANT: The concepts of these weeks are extremely important, and require very careful reading. 
+> Make sure you understand these concepts as well as you can, as they are crucial to programming and your ability to do well in our program!
+
+
+
+
 ### Here are resources that we like you to read as a preparation for the coming lecture:
 
 Please watch the following parts of the course, [Programming Foundations Fundamentals](https://www.lynda.com/Programming-Foundations-tutorials/Welcome/83603/90426-4.html) on Lynda.com (if you don't have access to Lynda yet ask Gijs):
+
     <br>4. Writing Conditional Code
     <br>5. Modular Code
     <br>6. Iteration: Writing Loops
     <br>7. More About Strings
     <br>8. Collections
     <br>11. When Things Go Wrong 
 
-## Read
-- [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/prototype)
-
-## From the book _A Smarter Way To Learn JavaScript_ please read:
-- 'Loops' of [_A Smarter Way To Learn JavaScript_](https://github.com/Sharique-Hasan/SaylaniBatch2-JavaScript/blob/master/A%20Smarter%20Way%20to%20Learn%20JavaScript.pdf) : Chapters 18-20
-- 'Functions' of [_A Smarter Way To Learn JavaScript_](https://github.com/Sharique-Hasan/SaylaniBatch2-JavaScript/blob/master/A%20Smarter%20Way%20to%20Learn%20JavaScript.pdf) : Chapters 35 - 38
-- Functions ~ http://eloquentjavascript.net/03_functions.html
-- 'Objects' of [_A Smarter Way To Learn JavaScript_](https://github.com/Sharique-Hasan/SaylaniBatch2-JavaScript/blob/master/A%20Smarter%20Way%20to%20Learn%20JavaScript.pdf) : Chapters 69 - 75
-- 'Conditions' of [_A Smarter Way To Learn JavaScript_](https://github.com/Sharique-Hasan/SaylaniBatch2-JavaScript/blob/master/A%20Smarter%20Way%20to%20Learn%20JavaScript.pdf) : Chapters 10 - 14
-- Program structure ~ http://eloquentjavascript.net/02_program_structure.html
+## Read the following articles that are very important to programming!
+- [Array](http://javascript.info/array)
+- [Loops](http://javascript.info/while-for)
+- [Functions](http://javascript.info/function-basics)
+- [Functions](http://eloquentjavascript.net/03_functions.html)
```

Comment:
Although these two Function links point to different URLs, in the formatted text you only see the word Functions repeated twice:

- Functions
- Functions